src/app/clusters/operational-credentials-server/operational-credentials-server.cpp

```diff
@@ -39,6 +39,8 @@
 #include <support/logging/CHIPLogging.h>
 #include <transport/AdminPairingTable.h>
 
+#include "dac_chain.h"
```

Comment:
Access to the DAC chain should be through some delegate, rather than hardcoded.

Examples will need to have the delegate API initialized to the correct implementation. The following procedures likely need to be platform specific, as they amount to provisioning data, which may be factory-time.

- Signing a payload with the device attestation private key
- Populating the DAC bytes in a buffer
- Populating the PAI bytes in a buffer (if present)
- Populating the CD bytes in a buffer (if present)
- Populating the firmware digest bytes in a buffer (if present)

None of the above can be "baked into" the SDK without an additional abstraction. Adding these abstractions may be a necessary precondition for this PR being able to land without requiring significant rework.

Reply:
This is addressed in another PR.